src/crab/splitdbm/split_dbm.cpp

```
@@ -0,0 +1,422 @@
+
+bool SplitDBM::is_top() const { return g_.is_empty(); }
+
+prevail::ExtendedNumber SplitDBM::get_bound(const VertId v, const Side side) const {
+    if (side == Side::LEFT) {
+        return g_.elem(v, 0) ? -prevail::Number(g_.edge_val(v, 0)) : prevail::MINUS_INFINITY;
+    } else {
+        return g_.elem(0, v) ? prevail::Number(g_.edge_val(0, v)) : prevail::PLUS_INFINITY;
+    }
+}
+
+void SplitDBM::set_bound(const VertId v, const Side side, const Weight& bound_value) {
```

Clang (14.0.6)

[warning] method 'set_bound' can be made static

(readability-convert-member-functions-to-static)